pkg/cluster/error.go

```diff
@@ -20,7 +20,7 @@ var (
 	errNoBackupExist     = errors.New("no backup exist for disaster recovery")
 	errInvalidMemberName = errors.New("the format of member's name is invalid")
 
-	errUnexpectedUnreadyMember = errors.New("unexpected unready member for selfhosted cluster")
+	errUnexpectedUnreadyMember = errors.New("unready member for selfhosted cluster")
```

Comment:
why it is not unexpected?

we do not really want to deal with that right now. we simply assume this shall ever happen.